Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -117,11 +117,9 @@ Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	Considered concussive (explosive) damage. Hits armor points 
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
-fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
-on stealth attacks. Reduces door-breaking DC by 30 per hit.
+	If armor roll is greater than 50, target rolls a DC 50 Fort save 
+for a broken limb of your choice. -20 to stealth rolls for being quiet 

```

Comment:
Is the Fort save to avoid the borked limb, keep composure at the borken ouch, smth else? We should be explicit abt the results of successful and failed saves.